part-4/modularization-concept.md

```diff
@@ -0,0 +1,248 @@
+
+- The equation in the **Kinetic** tab is overwritten.
+
+- The **Parameters** list is overwritten. This also implies that parameters that are not defined in the module that is lower in the hierarchy are removed.
+
+- The **Operator** (and/or) for the "Source" and "Target" lists are overwritten.
+
+- **Source** and **Target** lists are overwritten (i.e., it is possible to remove source/target condition).
+
+- **Include/Exclude** molecule lists for molecules are always extended. However, behavior of the **All checkbox** is overwritten.
+
+The current behavior of combining the passive transports might appear inconsistent and somewhat confusing. The discussion on this topic is still ongoing and can be followed [on GitHub](https://github.com/Open-Systems-Pharmacology/MoBi/issues/2053).
+
+#### Merge behavior "Extend"
+
+Currently, the merge behavior "Extend" for passive transports is identical to "Overwrite".
+
+### Observers
+
+##### Merge behavior "Overwrite"
+
+- The equation in the **Monitoring** tab is overwritten.
+
+- The **In container with** list is overwritten, including the **Operator** (and/or).
+
+- **Include/Exclude** molecule lists for molecules are always extended. However, behavior of the **All checkbox** is overwritten.
+
+The current behavior of combining the observers might appear inconsistent and somewhat confusing. The discussion on this topic is still ongoing and can be followed [on GitHub](https://github.com/Open-Systems-Pharmacology/MoBi/issues/2053).
+
+##### Merge behavior "Extend"
+
+Currently, the merge behavior "Extend" for observers is identical to "Overwrite".
+
+### Events
+
+Imagine the following case:
+
+1. Module `A` has an event `Event 1` and Module `B` has an event `Event 1`. The event from module `A` has the container criteria `Events`, and the event from module `B` has the container criteria `Organism`. The event from module `A` has a parameter `Param 1` and the event from module `B` has a parameter `Param 2`. 
+
+    When creating a simulation, these events are created separately based on their container criteria. Therefore, the simulation will contain two events called `Event 1` - one event in the node `Events` with the parameter `Param 1`, and one event in the node `Organism` with the parameter `Param 2`, disregarding the merge behavior.
+
+2. If the event from module `B` has container criteria `Events` OR `Organism`
+    - Merge behavior **Overwrite**: the final model will contain the `Event 1` in both containers `Events` and `Organism` with `Param 2` only (`Organism|Event 1` is defined in module `B` only, and `Events|Event 1` is defined in both modules and will be overwritten by the event defined in module `B`).
+    - Merge behavior **Extend**: the final model will contain the `Event 1` in container `Organism` with the parameter `Param 1`, and the event `Event 1` in container `Events` with parameters `Param 1` and `Param 2` (`Events|Event 1` is defined in both modules and will be extended).
+
```

Comment:
would change the order of created events in simulation to be consistent with other descriptions:

... the final model will contain the event `Event 1` in container `Events` with parameters `Param 1` and `Param 2` (`Events|Event 1` is defined in both modules and will be extended) and the `Event 1` in container `Organism` with the parameter `Param 1`.

Reply:
Done